joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/PocTest.scala

```diff
@@ -0,0 +1,172 @@
+package io.joern.rubysrc2cpg.querying
+
+import io.joern.rubysrc2cpg.testfixtures.RubyCode2CpgFixture
+import io.shiftleft.semanticcpg.language._
+
+class PocTest extends RubyCode2CpgFixture {
```

Comment:
Would be good to have a less generic name and rather point out what's being tested in the file name.

Reply:
Renamed to  IdentifierTests